Clients/src/presentation/components/ProjectCard/index.tsx

```diff
@@ -0,0 +1,68 @@
+import { Typography, Box } from "@mui/material";
+import Button from '@mui/material/Button';
+import { FC } from "react";
+import euimg from "../../assets/imgs/eu-ai-act.jpg"
+import ProgressBar from "./ProgressBar";
+import { styles } from "./styles";
+
+export interface ProjectCardProps {
+    id: number,
+    name: string,
+    owner_name: string,
+    last_update: string,
+    controls_completed: string,
+    requirements_completed: string
+}
+
+const ProjectCard: FC<ProjectCardProps> = ({
+    name,
+    owner_name,
+    last_update,
+    controls_completed,
+    requirements_completed
+}) => {
+    return (
+        <Box sx={styles.card}>
+            <Typography variant="h5" component="div" sx={styles.title}>
+                {name}
+            </Typography>
+            <Box sx={styles.upperBox}>
+                <Box>
+                    <Typography variant="subtitle1" component="span" sx={styles.subtitle}>
+                        Project owner
+                    </Typography>
+                    <Typography sx={styles.subtitleValue}>{owner_name}</Typography>
+                </Box>
+                <Box>
+                    <Typography variant="subtitle1" component="span" sx={styles.subtitle}>
+                        Last updated
+                    </Typography>
+                    <Typography sx={styles.subtitleValue}>{last_update}</Typography>
+                </Box>
+            </Box>
+            <ProgressBar progress={controls_completed} />
+            <Typography sx={styles.progressBarTytle}>
+                {controls_completed} controls completed
+            </Typography>
+            <ProgressBar progress={requirements_completed} />
+            <Typography sx={styles.progressBarTytle}>
+                {requirements_completed} requirements completed
+            </Typography>
```

Comment:
Nice component. I like it. 👍🏻
However, in the clean code principals, we have something called **Separation of Concerns**.
In this part, you can write a new helper function to avoid repeating yourself. Like this one:

```
    const renderProgressBar = (progress: string, label: string) => (
        <>
            <ProgressBar progress={progress} />
            <Typography sx={styles.progressBarTytle}>
                {progress} {label} completed
            </Typography>
        </>
    );
```

And then you can replace them with this:

```
            </Box>
            {renderProgressBar(controls_completed, "controls")}
            {renderProgressBar(requirements_completed, "requirements")}
            <Box sx={styles.lowerBox}>
```